_includes/markdown/Testing.md

```diff
@@ -45,6 +45,10 @@ cy.visit( `/wp-admin` );
 
 The purpose of E2E testing is to ensure the user-facing features work as expected. In the WordPress context, we can extend that purpose to "working as expected against supported WP versions and plugins/themes". At 10up, we're using GitHub Actions matrix and `wp-env` config override to solve that problem by [generating `wp-env` config](https://github.com/10up/simple-podcasting/blob/develop/tests/bin/set-core-version.js) for [each matrix](https://github.com/10up/simple-podcasting/blob/develop/.github/workflows/test-branch.yml#L30-L31). We're doing it for WP core version only, but it can be adapted and updated to handle more complex configurations.
 
+### Permalinks in multiple configurations
```

Comment:
@cadic We're having the permission issue with the custom WordPress versions rather than multiple configurations. Even if we test for a single configuration, we still have the permission issue. 

Maybe something like `Fixing the permission issue for custom WordPress versions` or `Fixing permalink issue for custom WordPress versions`?

Reply:
@dinhtungdu thank you, I agree the title needs update. I also removed reference to "WordPress core rather than latest" because looks like the issue is now happening on the default core as well: https://github.com/10up/restricted-site-access/actions/runs/1818649424